docs/diagrams/ArchiveSequenceDiagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+
+participant Logic
+participant ArchiveCommand
+participant Model
+participant "person:Person"
+
+Logic -> ArchiveCommand : execute()
+ArchiveCommand -> Model : getFilteredPersonList()
+ArchiveCommand -> "person:Person" : check if person is archived
```

Comment:
for this part should it be `isArchived()` since thats the command used to check if person is archived in `Person` object

Reply:
Yes you are correct